lib/s3mpi/interface.rb

```diff
@@ -37,10 +38,30 @@ def initialize _bucket, path = ''
     #    The number of times to attempt to store the object.
     def store(obj, key = SecureRandom.uuid, try = 1)
       s3_object(key).write(obj.to_json)
-    rescue 
+    rescue # This should really rescue specific errors
       (try -= 1) > 0 ? retry : raise
     end
 
+
+    # Load a CSV file, and store the contents in S3
+    # Proxies to store
+    #
+    # @param [String] csv_file_path path to the CSV file.
+    #    Passed to CSV.read
+    #
+    # @param [Hash] options Options hash.
+    #    Passed to CSV.read
+    def store_csv(csv_file_path, options = Hash.new)
+      store _load_csv(csv_file_path, options)
+    end
+
+    # Store a raw object
+    # @param obj [Object]
```

Comment:
Can you use consistent doc formatting? You have type/name above and name/type here.